charts/karpenter/templates/configmap-grafana-dashboards-karpenter-controllers-allocation.yaml

```diff
@@ -0,0 +1,352 @@
+{{- if .Values.grafana.dashboards.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboard-karpenter-controllers-allocation
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "karpenter.labels" . | nindent 4 }}
+  {{- with .Values.grafana.dashboards.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.additionalAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+data:
+  # https://github.com/aws/karpenter-provider-aws/blob/main/website/content/en/docs/getting-started/getting-started-with-karpenter/karpenter-controllers-allocation.json
+  karpenter-controllers-allocation.json: |
```

Comment:
I'm less familiar with the Grafana configuration, but do these need to be separate ConfigMaps or can they all be in the same ConfigMap but under different data keys?

Reply:
It's probably best to have them as separate config maps, due to the maximum object size and to reduce churn.